src/languages/en-US/commands/misc.json

```
@@ -1,4 +1,5 @@
 {
+	"birthdayNotConfigured": "A birthday channel or role has not been configured in this server. Configure one using the conf command.",
```

```suggestion
	"birthdayNotConfigured": "{{REDCROSS}} A birthday channel or role has not been configured in this server. You can configure both using the `{{prefix}}conf` command.",
```

And pass `commandContext.commandPrefix` to `{{prefix}}`